tests/components/modbus/test_modbus_sensor.py

```diff
@@ -24,195 +22,196 @@
     DATA_TYPE_INT,
     DATA_TYPE_STRING,
     DATA_TYPE_UINT,
+    MODBUS_DOMAIN,
 )
 from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
 from homeassistant.const import (
     CONF_ADDRESS,
     CONF_COUNT,
     CONF_DEVICE_CLASS,
+    CONF_HOST,
     CONF_NAME,
     CONF_OFFSET,
+    CONF_PORT,
     CONF_SENSORS,
     CONF_SLAVE,
     CONF_STRUCTURE,
+    CONF_TYPE,
     STATE_UNAVAILABLE,
 )
 from homeassistant.core import State
+from homeassistant.setup import async_setup_component
 
 from .conftest import ReadResult, base_config_test, base_test, prepare_service_update
 
 from tests.common import mock_restore_cache
 
 
 @pytest.mark.parametrize(
-    "do_discovery, do_config",
+    "do_config",
```

Comment:
hmm, looks like this should go away, or I'm missing something?

Reply:
Yes with 2021.7 do_discovery will always be true.

Reply:
nvm. since I removed do_discovery git diff did not recognize the whole thing and show the diff as deletion and addition, unfortunately. So I accidentally though that I re-added some deleted tests